src/webview/MessageList.js

```diff
@@ -113,45 +115,33 @@ export type Props = $ReadOnly<{|
 |}>;
 
 /**
- * A URL-like magic string denoting the webview-assets folder.
+ * The URL of the platform-specific assets folder.
  *
- * Not suitable for reuse or export.
+ * - On iOS: We can't easily hardcode this because it includes UUIDs.
+ *   So we bring it over the React Native bridge in ZLPConstants.m.
+ * - On Android: Different apps' WebViews see different (virtual) root
```

Comment:
```suggestion
 * - On iOS: We can't easily hardcode this because it includes UUIDs.
 *   So we bring it over the React Native bridge in ZLPConstants.m.
 *
 * - On Android: Different apps' WebViews see different (virtual) root
```